manifests/fpm/pool.pp

```diff
@@ -144,6 +148,7 @@
   $security_limit_extensions               = undef,
   $slowlog                                 = "/var/log/php-fpm/${name}-slow.log",
   $template                                = 'php/fpm/pool.conf.erb',
+  $process_dumpable                        = false,
```

Comment:
just a thought: does it make sense, in the matter of maintainability, to have every possible setting as parameter in the class? wouldn't it be better having a "custom code" parameter where the users can add whatever code they like?